### PR TITLE
[GITHUB-16] Update to Latest Standards

### PR DESCRIPTION
**Pull request by dennisconrad**
- Update molecule
- Update Boolean checks
- Fix Workspace User
- More extensive tests
- Support Ubuntu Bionic Beaver
- Support more versions

### CODE REVIEW

File: tasks/build.yml [hunk 1]
```diff
@@ -41,41 +40,48 @@
...
   lineinfile:
     create: yes
     dest: "/home/{{ sansible_nodejs_workspace_user }}/.bashrc"
     insertafter: EOF
     line: "export PATH=/home/{{ sansible_nodejs_workspace_user }}/.npm-global/bin:$PATH"
-  when: sansible_nodejs_workspace_user
+  when: sansible_nodejs_workspace_user is not none
 
 - name: Install NPMs
   become: yes
```
Comment (lobsterdore) [inline tasks/build.yml:76]: Would it make sense to run this as the workspace user if one has been specified?

Comment (lobsterdore) [reply tasks/build.yml:76]: Something like ```become_user: "{{ sansible_nodejs_workspace_user | default(omit) }}"``` might work

Comment (dennisconrad) [reply tasks/build.yml:76]: Turns out this is enough:

```yaml
become_user: "{{ sansible_nodejs_workspace_user }}"
```

**Review by lobsterdore — APPROVED** [review]
LGTM 👍

**Pull request merged**